apps/framr-web/lib/services/core/modules/rules/RuleInterface.ts

```diff
@@ -0,0 +1,16 @@
+import { CreateRule } from '../../../../types';
+
+export enum RulesEventChannel {
+  CREATE_RULES_CHANNEL = 'CREATE_RULES',
+  FIND_ONE_RULES_CHANNEL = 'FIND_ONE_RULES',
+  FIND_ALL_RULES_CHANNEL = 'FIND_ALL_RULES',
+  UPDATE_RULES_CHANNEL = 'UPDATE_RULES',
+  DELETE_RULES_CHANNEL = 'DELETE_RULES',
+}
+export interface RuleInterface {
+  create(RULES: CreateRule): void;
+  findOne(index: number): void;
```

Comment:
This will never work, I omitted to say that on previous PR, the id must be string